pkg/harvester/edit/loadbalancer.harvesterhci.io.loadbalancer/HealthCheck.vue

```diff
@@ -0,0 +1,120 @@
+<script>
+import { LabeledInput } from '@components/Form/LabeledInput';
+import LabeledSelect from '@shell/components/form/LabeledSelect';
+import RadioGroup from '@components/Form/Radio/RadioGroup';
+
+export default {
+  components: {
+    LabeledInput,
+    LabeledSelect,
+    RadioGroup,
+  },
+
+  props: {
+    value: {
+      type:     Object,
+      required: true,
+    },
+
+    model: {
+      type:     Object,
+      required: true,
+    },
+
+    mode: {
+      type:    String,
+      default: 'create'
+    },
+
+    disabled: {
+      type:    Boolean,
+      default: false,
+    },
+  },
+
+  data() {
+    return { healthCheckEnabled: !!this.value.port };
```

Comment:
Is it appropriate to use `this.value.port` to determine if healthCheckEnabled is enabled?

Reply:
Yes, port is required after enabled